api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -355,6 +355,12 @@ message RequestStateSetter {
   // Roles, if present, overrides the existing set of roles associated
   // with the access request.
   repeated string Roles = 6 [(gogoproto.jsontag) = "roles,omitempty"];
+  // AssumeTime is the time the requested roles can be assumed.
+  google.protobuf.Timestamp AssumeTime = 7 [
```

Comment:
nit: I find `AssumeTime` a little bit confusing. What do you think about `AssumeStartTime` or just `StartTime`

Reply:
Went with assumeStartTime as it seems clearer